code/modules/vehicles/unmanned/unmanned_vehicle.dm

```diff
@@ -57,8 +95,48 @@
 
 /obj/vehicle/unmanned/attackby(obj/item/I, mob/user, params)
 	. = ..()
-	if(!istype(I, /obj/item/uav_turret) && !istype(I, /obj/item/explosive/plastique))
+	if(istype(I, /obj/item/tool/wrench))
+		return remove_turret(user)
+	if(istype(I, /obj/item/uav_turret) || istype(I, /obj/item/explosive/plastique))
+		return equip_turret(I, user)
```

Comment:
iirc plastique has a isthing but I may be wrong

Reply:
nop